src/handwritten/_install_daqmx.py

```diff
@@ -238,14 +372,69 @@ def _install_daqmx_windows_driver() -> None:
 
     """
     installed_version = _get_daqmx_installed_version()
-    download_url, latest_version = _get_driver_details()
+    download_url, latest_version, release, supported_os = _get_driver_details("Windows")
+    if not download_url:
+        raise click.ClickException(f"Failed to fetch the download url.")
+    if not release:
+        raise click.ClickException(f"Failed to fetch the release version string.")
+    else:
+        if installed_version and latest_version:
+            _confirm_and_upgrade_daqmx_driver(
+                latest_version, installed_version, download_url, release
+            )
+        else:
+            _install_daqmx_driver_windows(download_url)
```

Comment:
Having a function named `_install_daqmx_windows_driver` call another function named `_install_daqmx_driver_windows` is too confusing. Likewise for `_install_daqmx_linux_driver` and `_install_daqmx_driver_linux`.

Rename them so that other developers can tell them apart. For example, it would be acceptable for `_install_daqmx_windows_driver` to call `_install_daqmx_windows_driver_core`, because that shows that the core implementation is in the 2nd function and the 1st function is just a wrapper.

Reply:
ok, it's still a [maze of twisty little passages](https://en.wikipedia.org/wiki/Colossal_Cave_Adventure#Other_media), but they are no longer all alike.